src/components/structures/MatrixChat.js

```diff
@@ -992,6 +985,13 @@ module.exports = React.createClass({
         }
     },
 
+    onRoomIdResolved: function(room_id) {
+        // It's the RoomView's resposibility to look up room aliases, but we need the
+        // ID to pass into things like the Member List, so the Room View tells us when
+        // its done that resolution so we can display things that take a room ID.
+        this.setState({currentRoomId: room_id});
```

Comment:
It should now keep the same one, since the room alias is preferred and this will stay the same.